### PR TITLE
Streamline installation

### PR DESCRIPTION
**Pull request by JakobPetersson**

### CODE REVIEW

**Comment by smuda** [comment]
My default hostname seems to be coming from DNS and is for example "master1.okd4.example.com". With the current script that creates the ignition path och http://infra1.okd4.example.com:8080/master1.okd4.example.com.okd4.example.com. I assume you have used simple hostnames in your setup?

File: README.md [hunk 1]
```diff
@@ -99,32 +99,40 @@ ansible-playbook -i hosts -v deploy-okd.yml --extra-vars "compliance_operator=tr
 Once the playbook tell you to, boot the masters on
 Fedora CoreOS USB and start the installation process. 
 The NUCs are a bit slow on the network side so a number of kernel arguments are needed.
+
+```shell
+# Automated version
```
Comment (smuda) [inline README.md:104]: Perhaps a different comment like "Use hostname from DHCP or DNS" is more appropriate?

File: README.md [hunk 2]
```diff
@@ -99,32 +99,40 @@ ansible-playbook -i hosts -v deploy-okd.yml --extra-vars "compliance_operator=tr
...
+$ bash -c "$(curl -fsSL http://infra1.okd4.example.com:8080/install.sh)"
+```
+
 ```shell
+# Manual version
```
Comment (smuda) [inline README.md:109]: Please harmonize this with the line above, for example:

# Manually set hostname
$ bash -c "$(curl -fsSL http://infra1.okd4.example.com:8080/install.sh)" -s master[1-3]

File: roles/2-wait-for-installation/tasks/main.yml [hunk 3]
```diff
@@ -14,20 +14,51 @@
...
+- name: Pause for starting CoreOS installation on masters
+  ansible.builtin.pause:
+    prompt: |
+      -----------------------------------
+      - Do the following on all masters -
+      -----------------------------------
+      1. Boot CoreOS
+      2. Run the following command to install CoreOS:
+
+      bash -c "$(curl -fsSL http://{{ groups['infras'][0] }}:8080/install.sh)"
```
Comment (smuda) [inline roles/2-wait-for-installation/tasks/main.yml:39]: Please add the possiblility to specify hostname in the instruction.

File: roles/2-wait-for-installation/tasks/main.yml [hunk 4]
```diff
@@ -14,20 +14,51 @@
...
-      Manually run sudo coreos-installer on bootstrap according to README.md and reboot.
-      Press return to continue
+      ---------------------------------
+      - Do the following on bootstrap -
+      ---------------------------------
+
+      1. Boot CoreOS
+      2. Run the following command:
+
+      bash -c "$(curl -fsSL http://{{ groups['infras'][0] }}:8080/install.sh)"
```
Comment (smuda) [inline roles/2-wait-for-installation/tasks/main.yml:57]: Does this work in your environment? Are you setting the hostname via DHCP as bootstrap?

**Review by smuda — CHANGES_REQUESTED** [review]
Update according to the comments.

**Comment by JakobPetersson** [comment]
Updated PR with your feedback.
Split the hostname input on '.' and take only the first part.
Updated comments and examples in README.

File: roles/2-wait-for-installation/tasks/main.yml (hunk 4 above)
Comment (JakobPetersson) [reply roles/2-wait-for-installation/tasks/main.yml:57]: Yes, the hostname is bootstrap and it works.

**Review by smuda — APPROVED** [review]

**Pull request merged**